packages/atlas/atlas.config.yml

```diff
@@ -2,6 +2,7 @@ general:
   appName: 'Atlas' # Application name - used in the copy throughout the app, in index.html, open graph meta tags, etc
   appDescription: 'The streaming platform empowering viewers, creators, and builders. Built on and operated by the Joystream blockchain and DAO.' # Application description - used in index.html meta tags
   appTagline: 'The streaming platform empowering viewers, creators, and builders. Built on and operated by the Joystream blockchain and DAO.'
+  appId: '$VITE_APP_ID' # App ID for Apps as first-class citizens
```

Comment:
the app breaks, because we didn't provide appId.
![image](https://user-images.githubusercontent.com/51168865/226923906-56104199-90c7-4078-adc6-68642282e44b.png)

Should we at least add `VITE_APP_ID` with a blank string in the env as a fallback?

Reply:
or maybe that should be null?